cellprofiler/setting.py

```diff
@@ -1709,12 +1709,24 @@ def shape(self, value):
 
     @property
     def size(self):
-        return int(self.value_text.split(",")[1])
+        _, size = self.value_text.split(",")
```

Comment:
Ah, I see. You use an empty string because you use `.split`.

It might make more sense to use `None` and check whether `value_text` exists.